media-libs/openjpeg/openjpeg-2.1.2.recipe

```diff
@@ -85,5 +86,7 @@ INSTALL()
 
 TEST()
 {
-	make check
+	cd tests
+	cmake ..
+	make
```

Comment:
**`make check`** did not work, so I changed it to something that does not fail... but it does not seem to launch any tests, unfortunately.
If anyone knows how to launch the tests on projects that use CMake that would be great. Thanks!